### PR TITLE
Add support for RCar-H3 board

### PR DESCRIPTION
**Pull request by OYTIS**
Both m3ulcb and h3ulcb rely on AGL BSP, there is currently no meta-updater-renesas layer.

h3ulcb wouldn't work right now until the respective PR to AGL is made.

### CODE REVIEW

File: classes/sota_h3ulcb.bbclass [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Commit united image to OSTree, not just uImage
+OSTREE_KERNEL = "Image"
+
+EXTRA_IMAGEDEPENDS_append_sota = " renesas-ota-bootfiles"
+IMAGE_BOOT_FILES_sota += "renesas-ota-bootfiles/*"
+
+OSTREE_BOOTLOADER ?= "u-boot"
+UBOOT_MACHINE_sota = "m3ulcb_defconfig"
```
Comment (eu-smirnov) [inline classes/sota_h3ulcb.bbclass]: should it be h3ulcb_defconfig?

File: classes/sota_m3ulcb.bbclass [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # Commit united image to OSTree, not just uImage
 OSTREE_KERNEL = "Image"
 
-EXTRA_IMAGEDEPENDS_append_sota = " m3ulcb-ota-bootfiles"
-IMAGE_BOOT_FILES_sota += "m3ulcb-ota-bootfiles/*"
+EXTRA_IMAGEDEPENDS_append_sota = " renesas-ota-bootfiles"
+IMAGE_BOOT_FILES_sota += "renesas-ota-bootfiles/*"
```
Comment (eu-smirnov) [inline classes/sota_m3ulcb.bbclass]: will this still be overridden by meta-agl/meta-agl-bsp/conf/include/agl_m3ulcb.inc?

File: classes/sota_h3ulcb.bbclass (hunk 1 above)
Comment (OYTIS) [reply classes/sota_h3ulcb.bbclass]: Yes, sure, thank you.

File: classes/sota_m3ulcb.bbclass (hunk 2 above)
Comment (OYTIS) [reply classes/sota_m3ulcb.bbclass]: If it is duplicated in `agl_m3ulcb.inc` it should be removed from there. PR to AGL is coming after this one is merged.

**Pull request merged**